src/components/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from "react";
+import {useForm} from "react-hook-form";
+
+function Form(){
+    const [agree, toggleAgree] = useState(false);
+    const { register,handleSubmit } = useForm();
+    function onFormSubmit(data){
+        console.log(data);
+    }
+    return(
+        <form onSubmit={handleSubmit(onFormSubmit)} >
```

Comment:
Hier zul je dan de functie moeten ontvangen en uitvoeren

```suggestion
function Form({ onSubmit }){
    const [agree, toggleAgree] = useState(false);
    const { register,handleSubmit } = useForm();
    function onFormSubmit(data){
        onSubmit();
        console.log(data);
    }
    return(
        <form onSubmit={handleSubmit(onFormSubmit)} >
```